### ISSUE TITLE
NZB download fails after '3.7.2rc1' to 'develop'

### ISSUE THREAD

**Issue by 8a8al00ey** [sabnzbd/sabnzbd#2426]
Start up logs.
https://paste.ee/p/hfkXh
tested: [5ccc124ad4d4d2120cb8ab86e34e6025d5c80011](https://hub.docker.com/layers/hotio/sabnzbd/nightly-5ccc124ad4d4d2120cb8ab86e34e6025d5c80011/images/sha256-edee296f3e5e728f8d6aefc9aa3307ae2e7a31cac53e2749b2bffa831bf4649d?context=explore)
[abb08a45898906e9f83bffa66b2cc5ea46e6a219](https://hub.docker.com/layers/hotio/sabnzbd/nightly-abb08a45898906e9f83bffa66b2cc5ea46e6a219/images/sha256-0bb8ae229fb3ce4270d62d2723952059a41e8c8f5f51f1dab2160a09592e84ec?context=explore)
[06ca51eabd19fdb1461ed19d758d6c87d4690aa3](https://hub.docker.com/layers/hotio/sabnzbd/nightly-06ca51eabd19fdb1461ed19d758d6c87d4690aa3/images/sha256-b239994aca0c4134470521d641f0c01cd7bb7411f6b33c8ccc6d3e93458370d9?context=explore)
[ba68243dc71ebb4e446fac601f3d8273d61d0ef3](https://hub.docker.com/layers/hotio/sabnzbd/nightly-ba68243dc71ebb4e446fac601f3d8273d61d0ef3/images/sha256-0a7c183f96213059b0c78260feaa834091cd7c9153d4a84248f82d353b018660?context=explore)

```
2023-01-26 11:31:53,917::INFO::[downloader:779] Connecting 5@news-eu.X.com finished
Exception in thread Thread-2:
Traceback (most recent call last):
  File "/app/sabnzbd/decoder.py", line 147, in run
    raise BadYenc
sabnzbd.decoder.BadYenc

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
Exception in thread Thread-3:
Traceback (most recent call last):
  File "/app/sabnzbd/decoder.py", line 147, in run
    raise BadYenc
sabnzbd.decoder.BadYenc

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/app/sabnzbd/decoder.py", line 190, in run
    if nzo.precheck and raw_data and raw_data[0].startswith(b"223 "):
AttributeError: 'int' object has no attribute 'startswith'
    self.run()
  File "/app/sabnzbd/decoder.py", line 190, in run
    if nzo.precheck and raw_data and raw_data[0].startswith(b"223 "):
AttributeError: 'int' object has no attribute 'startswith'
2023-01-26 11:31:53,922::INFO::[downloader:779] Connecting 11@news.X.eu finished
```

**Comment by Safihre** [comment]
Sorry about that.
We haven't fixed the "Check before download" yet. If you disable that it should work again.

**Comment by 8a8al00ey** [comment]
> Sorry about that. We haven't fixed the "Check before download" yet. If you disable that it should work again.

Worked. That got it going again.

**Comment by Safihre** [comment]
Fixed by #2440 !
